docs/how-to/use-rocm-agent-enumerator.rst

```diff
@@ -0,0 +1,281 @@
+.. meta::
+  :description: Install ROCmInfo
+  :keywords: install, rocminfo, AMD, ROCm
+
+
+Using ROCm agent enumerator
```

Comment:
Find this confusing. 
The title is about the agent enumerator - don't understand the relationship to ROCmInfo.

Reply:
Added a description. Fixed.